alpine-consul/root/etc/services.d/consul-template/finish

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/execlineb -S1
+
+# only tell s6 to bring down the entire container, if it isn't already doing so
+# http://skarnet.org/software/s6/s6-supervise.html
```

Comment:
@matthewvalimaki, we might even be able to add a `sleep 1` in the finish script so that s6 doesn't restart it so quickly.

I guess I'm just not 100% on the consul watch to wait for nodes. I definitely like it better than the shell loop. But it just feels like it is making the container `logic heavy`. But, without s6-rc, if that is what it takes, then that is what it takes I guess.

Anyway, let me know your final thoughts and if you think the container definitely needs the watch, then I'll get to merging it in.